tiledb/sm/array_schema/array_schema.cc

```diff
@@ -67,6 +68,8 @@ using namespace tiledb::common;
 
 namespace tiledb::sm {
 
+class MemoryTracker;
```

Comment:
This can't be right? You should have a `#include "tiledb/common/memory_tracker.h" in this file, not a forward declaration. Its probably that this used to work when `context_resources.h` had that include, but its since been removed to reduce compile times.

Reply:
Resolved